include/seqan3/search/all.hpp

```diff
@@ -20,34 +20,35 @@
  * component for searching large amounts of data and are used for tools such as read mappers, assemblers or protein
  * search tools.
  *
- * \if DEV
- * There are currently two major kind of indices: FM indices and k-mer indices (also known as q-gram
- * indices).
+ * \if KMER
+ * SeqAn currently implements two kinds of indices: FM indices and k-mer indices (also known as q-gram indices).
+ * Generally speaking, k-mer indices support very fast searching of exact k-mers (strings of length k) or k-mers with
+ * predefined wildcard positions, which match with any character.
+ * FM indices on the other hand are more versatile and work
+ * with arbitrary pattern lengths and error numbers or positions.
+ * \else
+ * SeqAn currently implements only the FM index and a k-mer index is planned. The FM index works
+ * with arbitrary pattern lengths and error numbers or positions.
```

Comment:
```suggestion
 * with arbitrary pattern lengths and error numbers.
```

The positions probably refers to "you can have an error anywhere in the query" ?
But I think it's rather confusing without additional explanation and would just remove it.

Reply:
Yes that was my intention...